src/main.rs

```diff
@@ -108,6 +116,10 @@ struct Opt {
     #[clap(long, env = "COMMA_PICKER", default_value = "fzy")]
     picker: String,
 
+    /// Nix index database location
+    #[clap(required = false, long, env = "NIX_DATABASE")]
```

Comment:
It would be better to implement this in nix-index.

there's a pr switching nix-index to use clap https://github.com/bennofs/nix-index/pull/186, so after that is merged this can be implemented in there

if this is implemented here then it should be a better variable name that can be reused in nix-index so we can remove it here, and there should be no -option

Reply:
You mean the env variable? I agree with that but shouldnt adding an additional cmdline option make comma more ergonomic?

Reply:
Hmm we already have --picker so --database should be fine

Reply:
Okay, I have updated the pull request to just add the new argument:)

Reply:
I mean't to have both the variable and the argument

Reply:
My bad, I have reverted the change:)